dialects/core/be/ll/ll.cpp

```diff
@@ -313,7 +305,27 @@ void Emitter::emit_epilogue(Lam* lam) {
             }
         }
     } else if (auto ex = app->callee()->isa<Extract>(); ex && app->callee_type()->is_basicblock()) {
-        emit_unsafe(app->arg());
+        // A call to an extract like constructed for conditionals (else,then)#cond (args)
+        // TODO: we can not rely on the structure of the extract (it might be a nested extract)
```

Comment:
If this is not the case, we should simply abort compilation; in addiont, we shouuld have a phase that splits nested extracts into basic blocks so the LLVM (or any other backend) doesn't have to fight this issue.

Reply:
I opened #187 and reverted the change.